scripts/colmap2nerf.py

```diff
@@ -43,6 +43,7 @@ def parse_args():
 	parser.add_argument("--out", default="transforms.json", help="Output path.")
 	parser.add_argument("--vocab_path", default="", help="Vocabulary tree path.")
 	parser.add_argument("--overwrite", action="store_true", help="Do not ask for confirmation for overwriting existing images and COLMAP data.")
+	parser.add_argument("--mask_categories", nargs="*", type=int, default=[], help="Object categories that should be masked out from the training images. See the category-to-id mapping here: https://gist.github.com/yenchenlin/75931e87cedad3d23868a79046fc97c2")
```

Comment:
Looking at the id->object mapping, the object names themselves seem fairly well understandable. Would it be possible to let the user pass a bunch of strings and only use the id->string mapping if a string happens to be a number?